src/sass/textbox/textbox.scss

```diff
@@ -193,6 +193,10 @@ textarea.textbox__control {
     }
 }
 
+.textbox--large {
+    height: 48px;
+}
+
 .textbox--large > input.textbox__control {
     height: 48px;
 }
```

Comment:
This is also a problem for `.textbox` (without `--large`). I like moving the size up to the parent container, but if you are doing that you should also change the child.

These are the changes I think we need:
- `.textbox { height: 40px }`
- `.textbox__control { height: 100% }`
- Remove this selector and its contents (`.textbox--large > input.textbox__control`)

Reply:
@LuLaValva , I reworked this based on our discussion.